dumpling/export/sql.go

```diff
@@ -353,7 +353,7 @@ func buildOrderByClause(conf *Config, db *sql.Conn, database, table string, hasI
 // SelectTiDBRowID checks whether this table has _tidb_rowid column
 func SelectTiDBRowID(db *sql.Conn, database, table string) (bool, error) {
 	const errBadFieldCode = 1054
-	tiDBRowIDQuery := fmt.Sprintf("SELECT _tidb_rowid from `%s`.`%s` LIMIT 0", escapeString(database), escapeString(table))
+	tiDBRowIDQuery := fmt.Sprintf("SELECT _tidb_rowid from `%s`.`%s` LIMIT 1", escapeString(database), escapeString(table))
```

Comment:
Why do we need to change it to `1`?

Reply:
Because I found that limit 0 can't explain the `row_id` status at some time.
```sql
mysql> SELECT _tidb_rowid FROM `INFORMATION_SCHEMA`.`CLUSTER_DEADLOCKS` limit 1;
ERROR 1105 (HY000): other error: Column ID -1 of table cluster_deadlocks not found
mysql> SELECT _tidb_rowid FROM `INFORMATION_SCHEMA`.`CLUSTER_DEADLOCKS` limit 0;
Empty set (0.00 sec)
```

Reply:
I think this should be a bug. 🥲

Reply:
Dumpling need to keep compatible with this to avoid failure.